twd.htm

```diff
@@ -167707,55 +167698,55 @@ <h3><a id="Year1997">1997</a></h3>
 2 Blood Doll
 1 Dreams of the Sphinx
 2 Memories of Mortality
-2 Perfectionist  (weren't too useful - the Vamps didn't act too
-often.  Maybe -1, +1 Blood Doll)
+2 Perfectionist  /* weren't too useful - the Vamps didn't act too
+often.  Maybe -1, +1 Blood Doll */
 1 Secure Haven
 1 The Barrens
 1 The Church of Vindicated Faith
 
 Event (4)
-1 Anthelios, the Red Star (would be another Unmasking if I owned one)
+1 Anthelios, the Red Star -- would be another Unmasking if I owned one
 2 Edge Explosion
 1 The Unmasking
 
 Actions (4)
-2 Psychic Veil (nice, but tough to do unless I had 2 vamps out)
-2 Scrounging (solid.)
+2 Psychic Veil -- nice, but tough to do unless I had 2 vamps out
+2 Scrounging -- solid.
 
 Action modifier (14)
 8 Cloak the Gathering
 6 Veil the Legions
 
 Ally (3)
 1 Carlton Van Wyk
-1 Moise Kasavubu (seems good, but never got him out to
+1 Moise Kasavubu /* seems good, but never got him out to
 confirm - in past uses, he's just gotten
-blocked, but with the stealth I put him back in)
-1 Vagabond Mystic (undecided; life was rarely an issue, except when
-a little healing wouldn't have mattered)
+blocked, but with the stealth I put him back in */
+1 Vagabond Mystic /* undecided; life was rarely an issue, except when
+a little healing wouldn't have mattered */
 
 Equipment  (16)
-5 Flash Grenade (could use more)
-1 Deer Rifle (I expected transient maneuvers, and wasn't wrong)
-2 .44 Magnum (could go away for Flash Grenades)
+5 Flash Grenade -- could use more
+1 Deer Rifle -- I expected transient maneuvers, and wasn't wrong
+2 .44 Magnum  --could go away for Flash Grenades
 1 Heart of Nizchetus
 1 Ivory Bow
 2 Laptop Computer
-2 Leather Jacket (because I own no NRA PAC)
+2 Leather Jacket -- because I own no NRA PAC
 1 Orb of Ulain
-1 The Crusader Sword (in retrospect, IR Goggles to go with
-this might be nice)
+1 The Crusader Sword -- in retrospect, IR Goggles to go with
+this might be nice
```

Comment:
Why isn't this includedd witht eh previous line as a /* .. */ style comment as used elsewhere?